Header header

float32 rho
float32 theta

geometry_msgs/Point sp
geometry_msgs/Point ep
geometry_msgs/Point raw_sp
geometry_msgs/Point raw_ep

int32 num
float32 sum_xx
float32 sum_yy
float32 sum_xy
float32 x_bar
float32 y_bar